skill.json

```diff
@@ -9,10 +9,7 @@
     "warning": "",
     "systemDeps": false,
     "requirements": {
-        "python": [
```

Comment:
Just a note the skill.json parsing (automation uses `neon-utils`) should be updated to parse from `setup.py` or something instead of `requirements.txt`

Reply:
Ah, and actually that's not going to work anymore either, since Poetry handles most of that